Build/libHttpClient.Android/app/src/main/java/com/xbox/httpClient/HttpClientRequest.java

```diff
@@ -0,0 +1,80 @@
+package com.xbox.httpclient;
```

Comment:
I'm not sure build is the right place to put source code
is it because the build files have to be in the same folder?
in that case it may be better to have those in the source

Reply:
Yeah, it's because for a Gradle project it expects the code to be in subdirectories. I also think it's a bit weird, but FWIW the current libHttpClient Java code also lives under `Build`, in `Build/libHttpClient.14(1|2).Android.Java/app`.

So I was mostly just following the convention there.

Reply:
It also, stupidly, means that there are now three copies of the Java code for libHC (one in the 141, 142, and this build def) in the repo. Fortunately those basically never change, but it isn't great. Not sure what the best way otherwise to organize it would be though.

Reply:
I really dislike this, but I guess it's fine for now